tsconfig.json

```diff
@@ -9,6 +9,7 @@
     "allowJs": false,
     "skipLibCheck": true,
     "esModuleInterop": true,
+    "noUnusedLocals": false,
```

Comment:
why?

Reply:
when you change it to true it raise an error on unused variables

Reply:
ok so i remove it.
stay only with linter.
then node modules not make a problem